src/presentational/ClassroomsOverview.jsx

```diff
@@ -16,7 +18,9 @@ const ClassroomsOverview = (props, context) => (
                 <i className="fa fa-institution fa-4x"></i>
               </div>
               <div className="col-xs-9 text-right">
+                {(context.classrooms.data.length > 0) ?
                 <h1>{context.classrooms.data.length}</h1>
```

Comment:
@simoneduca , I need to check: if a new user accesses this page for the first time, won't `context.classrooms.data.length` and `context.classrooms.members.length` both be 0? If yes, wouldn't new users always see the spinners, until they create a class and have a student?

Is there another way to check if the values have been properly loaded or not, perhaps `context.classrooms.loading === false`?